### ISSUE TITLE
PhantomJS support (partial)

### ISSUE THREAD

**Issue by TheFifthFreedom** [selenodium/selenodium-grid#39]
This is less of an issue and more of a suggestion, but I noticed by accident that PhantomJS, while supported natively by Selenium now, isn't supported by Selenodium yet:

```
[warn]: Cannot find capabilities for request (41711009):
{"browserName":"phantomjs","javascriptEnabled":true,"platform":"ANY","version":""}
```

I just thought it would make for a great addition in the future!

**Comment by arikon** [comment]
@TheFifthFreedom What do you mean by "PhantomJS  supported natively by Selenium now"? As one of the nodes?

**Comment by arikon** [comment]
I tried to investigate and found this:
1. PhantomJS 2.0.0 has a broken GhostDriver https://github.com/detro/ghostdriver/issues/394; this prevents it from registering to a hub (any, not just `selenodium-grid`)
2. PhantomJS 1.9.8 has a broken http-parsing layer https://github.com/ariya/phantomjs/issues/11421; this prevents it from receiving commands from the `selenodium-grid`

So the problem is on the PhantomJS side.

**Comment by arikon** [comment]
I've also pushed a fix that prevented PhantomJS 1.9.x from working at all (it was disconnected after some time): 228912d4c3756ac6ceaceb31b84f07f75e95d2dc

**Comment by TheFifthFreedom** [comment]
Good point: Ghost Driver updates seem to take time before they're back ported into their respective PhantomJS versions, but it looks like the current state of things is quite poor, and as you say if anything has to happen, it would be on PhantomJS' side. Here's hoping they will fix this eventually. Thanks for looking into this!
